microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -64,8 +64,16 @@ With this release, updates for both minor releases and patch releases are suppor
 //[id="microshift-4-14-support"]
 //=== Support
 
-//[id="microshift-4-14-post-installation"]
-//=== Post-installation configuration
+[id="microshift-4-14-post-installation"]
+=== Post-installation configuration
+
+[id="microshift-4-14-changes-config-yaml"]
+==== Configuration options changes
+* File placement for a custom YAML is now only `/etc/microshift/config.yaml`. `$HOME/.microshift/config.yaml` is no longer used.
+* The field name `CIDR` for grouping IP addresses was removed.
```

Comment:
This line need some more detail to explain which of several places a CIDR value is used was changed. Here's some suggested wording:

The clusterNetwork field format has changed from a list of structures with a `cidr` field to a list of strings containing CIDR values.